### DISCUSSION TITLE
Any way to init db programmatically when using embedded sqlite?

### DISCUSSION THREAD

**Discussion by bkkgbkjb** [mikro-orm/mikro-orm#3914] [Q&A]
Hi all,

thanks in advance for any help and if answer exists for my question, please direct me there

I'm currently writing something in `electron` and tries to embed a `sqlite` db inside my app

Question is:
as a desktop app with embedded db, I need to consider situation when users first run my app and init all things done

In this case, how could I use `mikro-orm` to populate my empty db (based on `entities` I defined in `mikro-orm` way)?

I know I could directly write `SQLs` into `sqlite`, but that is not `DRY` as I don't use my `entities` defined already`

Thanks

**Comment by B4nan** [comment]
You could run `orm.schema.updateSchema()` after your app starts.

https://mikro-orm.io/docs/schema-generator#using-schemagenerator-programmatically
